nix/cells/marconi/library/cabal-project-index-state.nix

```diff
@@ -2,4 +2,4 @@
 
 { inputs, cell }:
 
-cell.library.marconi-project.index-state
+cell.library.marconi-project.index-state-max
```

Comment:
What is that `index-state-max`?

Reply:
They have renamed `index-state` to `index-state-max` in haskell-nix.